client/tests/test_source_dataframe.py

```diff
@@ -28,9 +34,14 @@ def test_dataframe_for_name_variant_args(
     transformation = arrange_transformation(provider, is_local)
 
     client = ff.Client(local=is_local, insecure=is_insecure)
-    client.apply(asynchronous=True)
+    # If we're running in a hosted context, `apply` needs to be synchronous
```

Comment:
Do we have a check or warning for this when running manually?

Reply:
What do you mean "running manually"? In any case, you need to apply in the context of the tests. And actually, now that I'm seeing this again, I don't even need to pass this arg because `asynchronous=True` is the default.

Reply:
Just wondering if someone decides to us it and run it async=False, is there a check for it, or will it just error?

Reply:
There's not check, so I believe the behavior is it'll error